resources/web/wwi/Toolbar.js

```diff
@@ -10,13 +10,17 @@ export default class Toolbar {
     this._view = view;
     this.type = type;
     this.parentNode = parentNode;
+    this.size = 0;
```

Comment:
Using `width` instead of `size` would be more precise, isn't it?
```suggestion
    this.width= 0;
```

Reply:
Or even better `minWidth`?